addons/l10n_tr_nilvera/models/res_partner.py

```diff
@@ -13,7 +13,7 @@ class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner']
 
-    invoice_edi_format = fields.Selection(selection_add=[('ubl_tr', "Turkyie (UBL TR 1.2)")])
+    invoice_edi_format = fields.Selection(selection_add=[('ubl_tr', "Türkiye (UBL TR 1.2)")])
```

Comment:
This would need an update of the pot and po file :grimacing:, also even if it's a little change i would add it to the commit message, or even create a new commit for that change and the translation :smile: